components/director/internal/domain/application/resolver_test.go

```diff
@@ -968,41 +968,60 @@ func TestResolver_Documents(t *testing.T) {
 		Name           string
 		ServiceFn      func() *automock.DocumentService
 		ConverterFn    func() *automock.DocumentConverter
-		InputFirst     *int
-		InputAfter     *graphql.PageCursor
+		PersistenceFn    func() *persistenceautomock.PersistenceTx
+		TransactionerFn  func(persistTx *persistenceautomock.PersistenceTx) *persistenceautomock.Transactioner
 		ExpectedResult *graphql.DocumentPage
 		ExpectedErr    error
 	}{
 		{
 			Name: "Success",
+			PersistenceFn: func() *persistenceautomock.PersistenceTx {
+				persistTx := &persistenceautomock.PersistenceTx{}
```

Comment:
probably not covered cases:
- error on commit
- error on starting a transaction

Reply:
As discussed, we will migrate to new approach in #257, so that's why I won't add these test cases